<launch>
  <param name="use_sim_time" value="false"/>
  <!-- <node pkg="bot_sim" type="lidar_filter" name="lidar_filter" output="screen">
      <param name="subscribed_scan" type="string" value="scan_laser" />
      <param name="new_scan" type="string" value="new_scan" />
      <param name="min_angle" type="double" value="-1.2" />
      <param name="max_angle" type="double" value="1.2" />
  </node> -->
  <node pkg="bot_sim" type="threeD_lidar_merge_pointcloud" name="threeD_lidar_merge_pointcloud" output="screen">
      <param name="laser_frame" type="string" value="aft_mapped" />
      <param name="scan_topic_left" type="string" value="/livox/lidar_192_168_1_3" />
      <param name="scan_topic_right" type="string" value="/livox/lidar_192_168_1_141" />
      <param name="new_scan_topic" type="string" value="/3Dlidar" />
  </node>
</launch>
